crt_portal/cts_forms/model_variables.py

```diff
@@ -153,17 +153,22 @@
     ('VOT', _('Voting')),
 )
 
-PLACE_CHOICES = (
-    ('home', _('Home, potential home, or services to help with purchasing a home (banks, lenders, or other financial services)')),
-    ('workplace', _('Workplace or potential workplace')),
-    ('school', _('Educational institution (school, university), education program or educational activity (after school program or workshop)')),
+COMMERCIAL_OR_PUBLIC_PLACE_CHOICES = (
     ('place_of_worship', _('Place of worship')),
-    ('store', _('Retail/commercial building (store, restaurant, hotel, nightclub, theater, gym, or other commercial space)')),
-    ('public_space', _('Outdoor public space (including car, street, sidewalk, park)')),
-    ('voting', _('Voting location or ballot (including mail-in ballots)')),
-    ('healthcare', _('Healthcare facility (including mental health or long-term care)')),
-    ('incarcerated', _('Prison, jail, or juvenile corrections facility, or while otherwise incarcerated')),
-    ('government_building', _('Another government building (courthouse, DMV, post office)')),
+    ('store', _('Commercial or retail building')),
+    ('healthcare', _('Healthcare facility')),
+    ('financial', _('Financial institution')),
+    ('public_space', _('Public outdoor space')),
```

Comment:
[content doc](https://docs.google.com/document/d/1bxSRc9CQOHjQhTaYkHncE4tykUEFeC6RzFtP5IQk0Ic/edit?ts=5de6f69e#) says "Public space"

Reply:
This must have been updated in the last couple of days, definitely checked there. Thanks for double checking

Reply:
No worries, the content does change